documentation/common/overview.adoc

```diff
@@ -0,0 +1,31 @@
+[[enmasse_overview]]
+
+
+EnMasse is an open source project for managed, self-service messaging on [Kubernetes](https://kubernetes.io). EnMasse can run on your own infrastructure or in the cloud, and simplifies running a messaging infrastructure for your organization.
+
+The *service admin* can deploy and manage messaging infrastructure, while *tenants* can request messaging resources, both using cloud-native APIs and tools.
+
+=== Features
+
+* Built-in *authentication* and *authorization* of clients and identity management
+* Runs on *Kubernetes*: deploy *on-premise* or in the *cloud*
+* Different *messaging patterns* like request-response, pub-sub and events
+* Decouple *operation* of infrastructure from configuration and use by *applications*
+
+EnMasse can be used for many purposes, such as moving your messaging infrastructure to the cloud *without depending on a specific cloud provider*, building a scalable messaging backbone for *IoT*, or just as a *cloud-ready* version of a message broker.
+
+EnMasse can provision different types of messaging depending on your use case. A user can request messaging resources by creating an <<address_space>>.
+
+EnMasse currently supports a `standard` and a `brokered` address space types, each with different semantics. 
+
+== Standard address space
+
+The standard address space type is the default type in EnMasse, and is focused on scaling in the number of connections and the throughput of the system. It supports AMQP and MQTT protocols, with more to come in the future. This address space type is based on based on other open source projects like [Apache ActiveMQ Artemis](https://activemq.apache.org/artemis/) and [Apache Qpid Dispatch Router](https://qpid.apache.org/components/dispatch-router/index.html) and provides elastic scaling of these components.  From 40000 feet, the messaging infrastructure looks like this: 
```

Comment:
Delete second "is based on"
Change "like" to "such as"

Instead of "From 40000 feet the messaging infrastructure looks like this:" suggest something like "This image illustrates the high-level architecture of the standard address space:"
Also maybe add a paragraph break directly before this sentence